Snakefile

```diff
@@ -148,7 +150,8 @@ rule plot_ecosystems:
 
 rule plot_contributions:
     input:
-        "git-log/parsed-log.tsv"
+        tsv="git-log/parsed-log.tsv",
+        script='scripts/plot-contributions.py'
```

Comment:
Why this? If everything with changed scripts shall be rerun, you can use `snakemake -R $(snakemake --list-code-changes)`.

Reply:
Ah, embarrassingly I didn't know that `--list-code-changes` covered `script:` directives too!

Reply:
Thanks to Andreas Wilm. He implemented that in the hackathon that took place in Berlin last winter ;-). But this is not very visible in the docs. I should improve that.